examples/simple/src/main.rs

```diff
@@ -18,7 +18,7 @@ mod tests {
 lazy_static! {
     static ref MIGRATIONS: Migrations<'static> =
         Migrations::new(vec![
-            M::up(include_str!("friend_car.sql")),
+            M::up(include_str!("../../friend_car.sql")),
```

Comment:
While I’m rebasing, I’ll move that file in the simple example folder.

Reply:
Hum on a second thought, it’s used in integration tests as well, let’s leave it there.